app/resources/api/rest/admin/package_counter_resource.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class Api::Rest::Admin::PackageCounterResource < ::BaseResource
+  model_name 'Billing::PackageCounter'
+
+  attribute :duration
+  attribute :exclude
+  attribute :prefix
+  attribute :service_id
+
+  paginator :paged
+
+  has_one :account, class_name: 'Account', exclude_links: %i[self]
+  has_one :service, class_name: 'Service', exclude_links: %i[self]
+
+  filter :'service.id', verify: :verify_service_id, apply: ->(records, value, _options) { records.where(service_id: value) }
```

Comment:
better to use relationship_filter there - in same way as we have in other resources

Reply:
also we need filter by account